src/components/tag/tag.tsx

```diff
@@ -1,20 +1,20 @@
-import { ReactNode } from 'react';
 import classNames from 'classnames';
 import styles from './tag.module.scss';
 
 type Props = {
-  children: ReactNode;
+  id: string;
   type?: 'primary' | 'secondary';
 };
 
-const Tag = ({ children, type = 'primary' }: Props) => (
+const Tag = ({ id, type = 'primary' }: Props) => (
```

Comment:
It makes sense to rename this component to `ProposalTag` now (remember the folder too).

Reply:
Yeah, definitely. Done.